core/client/tests/acceptance/settings/navigation-test.js

```diff
@@ -76,8 +74,14 @@ describe('Acceptance: Settings - Navigation', function () {
         application = startApp();
         store = application.__container__.lookup('store:main');
         server = new Pretender(function () {
+            // TODO: This needs to either be fleshed out to include all user data, or be killed with fire
```

Comment:
This and the fact that each authenticated app load will hit the notifications end-point was what led me to look into ember-cli-mirage again, I think we'll need to either switch to that or find some way of providing fixture data and default endpoints to all pretender instances used in acceptance tests.